src/main/cli/cli.c

```diff
@@ -4734,7 +4771,7 @@ static void cliStatus(const char *cmdName, char *cmdline)
 #ifdef USE_GPS
     cliPrint("GPS: ");
     if (featureIsEnabled(FEATURE_GPS)) {
-        if (gpsIsHealthy()) {
+        if (gpsData.state > GPS_STATE_CHANGE_BAUD) {
```

Comment:
Should we be exposing the internal GPS states to other modules?
Perhaps we can add status functions to the public GPS interface that answer these questions, such as isGpsConnected( ) and isGpsConfigured( ) and then let the GPS code check what it needs to, encapsulated within that module?
That way the other modules won't be coupled to that internal info and won't risk breaking if we change the state machine around.

Reply:
Noted, this kind of thing should be refactored in a later PR

Reply:
I won't be supporting this, I think it is clearer to use > STATE CHANGE BAUD to indicated that baud rate has been set, rather than >= CONFIGURE.

Reply:
However, you are very persistent :-)